src/main/java/com/fcgl/madrid/points/controller/UserPointsController.java

```diff
@@ -14,13 +22,22 @@
 public class UserPointsController {
     private UserPointsService userPointsService;
 
+    DevService devService;
+
+    @GetMapping("/userPoints")
+    public UserPoint getUserPointsById(Long userId) {
```

Comment:
You're going to want to change this return type to be the POJO class that `getUserPointsById` returns once you update this PR. The same applies to the other endpoint you added for UserTrophies